/**:
  ros__parameters:
    w_rot: -0.01248437168647055
    x_rot: 0.6995545218795377
    x_trans: 0.028555265326616244
    y_rot: 0.7144281747637923
    y_trans: -0.51286341187231
    z_rot: -0.0074403672201773284
    z_trans: 0.7890453972078894
